runtime/onert/core/src/loader/CircleLoader.cc

```diff
@@ -299,6 +307,69 @@ void CircleLoader::loadCall(const Operator *op, ir::Graph &subg)
   subg.addOperation(std::move(new_op));
 }
 
+void CircleLoader::loadRunModel(const Operator *op, ir::Graph &subg)
+{
+  ir::OperandIndexSequence inputs;
+  ir::OperandIndexSequence outputs;
+
+  loadOperationIO(op, inputs, outputs);
+
+  // As workaround, we assume that only tvn type model is used for RUN_MODEL
+  // So subgraph will have only one OP: Bulk
+  // We copy this OP into this subgraph (like model inlining)
+  //
+  // TODO Introduce RunModel operator type to support various model
+  // TODO Introduce ModelIndexManager to handle various case
+  //      - ModelIndex mapping for nnpackage's multimodel
+  //      - ModelIndex mapping for RUN_MODEL
+  //      - Lazy loading for model index for RUN_MODEL
+  //      - Model inlining on optimize phase (compile)
+  assert(!_file_path.empty());
+  auto model_base_path = std::filesystem::path(_file_path).parent_path();
+  // TODO Get location from BuiltinOptions instead of custom OP option
+  auto location = getCustomOpAttrMap(op)["location"].ToString();
+  if (location == "null")
+    throw std::runtime_error("Cannot find 'location'");
```

Comment:
When map is not exists, flexbuffer return empty map, and when we try to find value from empty map or not existing key, it return Reference object including null flatbuffer type (not nullptr), and using ToString() to null flatbuffer type object return "null" string. So we can check existence of map and "location" key by checking string value is "null" or not.